### PR TITLE
Clarify wait behavior and element method behavior

### PR DESCRIPTION
**Pull request by asavageiv**
Clarify that displayed? waits for the element to appear according to Capybara's default wait time by default and can be given a wait time argument.

Clarify that the element finding methods delegate to Capybara methods.

This will make it easier for users to understand the behavior of SitePrism as it relates to the underlying Capybara behavior and can take advantage of improvements in Capybara's documentation.

### CODE REVIEW

**Comment by asavageiv** [comment]
Fixes #73

File: README.md [hunk 1]
```diff
@@ -476,6 +485,8 @@ Then(/^the search field exists$/) do
 end
 ```
 
+The method delegates to [Capybara::Node::Matchers#has_selector?](https://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Matchers#has_selector%3F-instance_method)
```
Comment (luke-hill) [inline README.md:488]: Move this to line 462 (After the name) - Also rename the start to `This method`

Same for all other instances below

**Review by luke-hill — COMMENTED** [review]
Good stuff. Couple of quick changes to make.

**Review by luke-hill — COMMENTED** [review]
Can you add a generic changelog entry about updating the readme to improve documentation. See other posts in previous versions.

I'll then merge this in :D

**Comment by asavageiv** [comment]
Done. Sorry for being slow.

**Pull request merged**